index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title> home </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./css/styles.css"/>
+    </head>
+    <body>
+        <header>
+            <div class="brand">
+                <a class="logo" href="#"> My Logo </a>
+            </div>
+            <nav>
+                <div>
+                    <img src="./images/dashicons_menu.png" alt="dashicon menu icon"/>
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <div class="title">
+                <h1> I’m Mariane Glad to see you! </h1>               
+            </div>
+            <div>
+                <p class="headline">
+                    I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div>
+                <h2 class="connect"> LET’S CONNECT </h2>
+            </div>
+            <ul class="icons">
+                <li>
+                    <a href="#">
+                        <img src="./images/twitter.svg" alt="twitter logo"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Linkedin_icon.svg" alt="linkedin logo"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/medium.svg" alt="medium logo"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/github.svg" alt="github logo"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Vector.svg" alt="vector logo"/>
+                    </a>
+                </li>
+            </ul>
```

Comment:
- [ ] Kindly make sure the icons are well aligned as seen in the original template😉✌
- Figma design
![screenshot (3)](https://user-images.githubusercontent.com/44624138/143332077-9791a578-b9ae-4d5b-a965-7e353a12db5e.png)
- Your design
![local design](https://user-images.githubusercontent.com/44624138/143332293-c8df1a92-cd23-407a-ade2-d461f25bf778.jpg)